### ISSUE TITLE
SwipeableListItem onClick action triggered, when element swiped 

### ISSUE THREAD

**Issue by canberkakartuna** [marekrozmus/react-swipeable-list#35]
Swipe action triggers onClick event

1. Swipe trailing actions
2. 'SwipeableListItem' element onClick triggered

Expected behavior:
onClick should work apart from swipe actions

**Desktop (please complete the following information):**
 - OS: iOS
 - Browser Safari

Code:

```
    const trailingActions = (onDelete) => (
      <TrailingActions>
        <SwipeAction onClick={onDelete} destructive={true}>
          <ActionContent style={{ backgroundColor: 'red' }}>
            <DeleteOutlined style={{ fontSize: 18 }} />
          </ActionContent>
        </SwipeAction>
      </TrailingActions>
    );

    <SwipeableList fullSwipe={false} type={ListType.IOS}>
      {!!items.length &&
        items?.map((item) => (
          <SwipeableListItem
            leadingActions={null}
            trailingActions={trailingActions(() => removeItem(item))}
            key={item?.id}
            onClick={(e) => {
              console.log('click'); // THIS CODE SHOULD NOT WORK ON SWIPE BUT IT DOES.
            }}
          >
            <Card style={{ borderRadius: 0, width: '100%' }}>
              <Row>
                <Col span={1}>1</Col>
                <Col span={20}>
                  <p>{item?.name}</p>
                </Col>
                <Col span={3}>
                  <p>{item?.totalCost}</p>
                </Col>
              </Row>
            </Card>
          </SwipeableListItem>
        ))}
    </SwipeableList>
```

**Comment by marekrozmus** [comment]
@all-contributors please add @canberkakartuna for bug

**Comment by marekrozmus** [comment]
@canberkakartuna Fixed in 1.9.1 - thanks :)

**Comment by canberkakartuna** [comment]
Thanks @marekrozmus :)
